src/Aop/Pointcut/ClassDeclaresPointcut.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/*
+ * Go! AOP framework
+ *
+ * @copyright Copyright 2016, Lisachenko Alexander <lisachenko.it@gmail.com>
+ *
+ * This source file is subject to the license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Go\Aop\Pointcut;
+
+use Go\Aop\Pointcut;
+use Go\Aop\PointFilter;
+
+/**
+ * Pointcut that filters out all class method and properties which are not declared within class
+ */
+class ClassDeclaresPointcut implements Pointcut
```

Comment:
Idea is to have a pointcut matcher that will filter out all non-declared members and properties that are matched.